.travis-script.sh

```diff
@@ -4,26 +4,38 @@ if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; th
   # Handle master builds
   echo "Building Docker image for ${TRAVIS_BRANCH}"
   grunt build
-  docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD" tutum.co
-  docker build -t tutum.co/$DOCKER_USERNAME/angular-cart-demo:latest .
-  docker push tutum.co/$DOCKER_USERNAME/angular-cart-demo:latest
+  docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD" $DOCKER_REGISTRY
+  docker build -t $DOCKER_REGISTRY/$DOCKER_USERNAME/angular-cart-demo:latest .
```

Comment:
needs change to check for the existence of ```$DOCKER_REGISTRY``` otherwise push to default reg (i/e. dockerhub)